genomic_data_service/region_indexer_task.py

```diff
@@ -6,10 +6,14 @@
 from genomic_data_service.parser import SnfParser, RegionParser
 from genomic_data_service.constants import DATASET
 import uuid
+from os import environ
 
 
 celery_app = Celery('regulome_indexer')
-celery_app.config_from_object('config.celeryconfig')
+if "DOCKER" in environ:
+    celery_app.config_from_object('config.celeryconfig_docker')
+else:
+    celery_app.config_from_object('config.celeryconfig')
```

Comment:
I think this is also possible to be passed explicitly. https://stackoverflow.com/questions/35674946/how-to-get-celery-to-load-the-config-from-command-line should be a good start for figuring this out.